src/modules/library/containers/LibraryContainer.js

```diff
@@ -227,36 +229,50 @@ function LibraryContainer(props) {
 										</>
 									)}
 								</div>
-								{activeNode.name && isReadOnly && (
-									<Button
-										size="sm"
-										color="secondary"
-										className="mr-2"
-										onClick={() => setReadOnly(false)}
-									>
-										{t("ASABLibraryModule|Edit")}
-									</Button>
-								)}
-								{activeNode.name && !isReadOnly && (
-									<div>
+								<ButtonGroup>
+									{activeNode.name && isReadOnly && (
 										<Button
```

Comment:
@titkov49 @ateska Should not be Editing of the files in library restricted by some resource?